tslib/src/functions/util-submit-check/fake-check-audit-generator.service.ts

```diff
@@ -0,0 +1,129 @@
+import { CheckQuestion, CompleteCheckAuditEntry, AuditEntryType } from '../../schemas/check-schemas/submitted-check'
+import moment from 'moment'
+
+export class FakeCheckAuditGeneratorService {
+  createAudits (questions: CheckQuestion[]): CompleteCheckAuditEntry[] {
+    return [...this.buildWarmupEntries(), ...this.buildQuestionEntries(questions), ...this.buildSubmissionEntries()]
+  }
+
+  private buildSubmissionEntries (): CompleteCheckAuditEntry[] {
+    return [
+      {
+        type: AuditEntryType.CheckSubmissionPending,
+        clientTimestamp: moment().toISOString(),
+        relativeTiming: '+0'
```

Comment:
You don't need this `relativeTiming` key - it's added to the checkViewer to make the timings more human-readable, but it's not actually part of the message spec.